puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/pipelines.groovy

```diff
@@ -0,0 +1,25 @@
+def installPipelineVars(version, os, type) {
+    playBook = [
+      'boxes': ["pipeline-${type}-server-${version}-${os}", "pipeline-${type}-smoker-${version}-${os}"],
+      'pipeline': 'install_pipeline.yml',
+      'extraVars': [
+        'pipeline_version': version,
+        'pipeline_os': os,
+        'pipeline_type': type
+      ]
+    ]
+    return playBook
+}
+
+def upgradePipelineVars(version, os, type) {
```

Comment:
you use `version, os, type` here, but all callers use `type, version, os`, which seems better to me?